gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -59,7 +59,7 @@
           </div>
         </div>
         <div ng-if="!editingDeviceDefaults()" id="device-sharing" class="tab-pane">
-          <div class="row">
+          <div class="row" ng-if="!currentDevice.untrusted"> <!-- The entire row with introducer and auto accepting folders disappear for untrusted devices. -->
```

Comment:
Yeah, I agree, and I did this out of laziness -- disabling them requires also unchecking them somewhere etc to make it look reasonable, while this is easier :)

Reply:
Did the unchecking: https://github.com/imsodin/syncthing/tree/untrusted

No styling as that turned out to be non-trivial (who would have guessed), but seems okish to me:
![Screenshot_2023-06-10_21-18-19](https://github.com/syncthing/syncthing/assets/15955093/54785763-5d32-481e-9cef-6bcb8bd237cd)

Reply:
Here's that fix: https://github.com/syncthing/syncthing/pull/8933

Reply:
Without styling they're not disabled though, they're just mysteriously nonfunctional / broken...

Reply:
Oh dang the curso didn't make it into the screenshot as planned: It becomes a "disabled sign" when hovering the checkbox. And if you look closely, the checkbox is grey not black :)

Reply:
Ok, I am raising the terrible UI stakes :)  
![Screenshot_2023-06-11_11-30-27](https://github.com/syncthing/syncthing/assets/15955093/682b5551-487f-4183-b60e-f4133cbbbc8d)

Reply:
Tooltip why?

Reply:
Happy to add a tooltip, however lets first here from @calmh: Is this thing is even something you'd considers over the hiding?

Reply:
Yeah, for sure, I didn't look at the code but it looks reasonable to me. Something like a `Disabled because Untrusted is set` as a tooltip would be awesome.

Reply:
Added tooltips. I was mainly asking because if you stare at it long enough, I don't think it looks entirely great (then again a few things don't in our UI). The code also isn't nice, but that's perfectly normal angularjs unpleasentness.